New `make report-*` errors since #1233

#1233 introduced errors in `make report-newlib`.

```
               ========= Summary of gcc testsuite =========
                            | # of unexpected case / # of unique unexpected case
                            |          gcc |          g++ |     gfortran |
 rv64imafdc/  lp64d/ medlow |19208 /  3644 | 9845 /  2462 |      - |
```

<details>
    <summary>Subset of unexpected fails list</summary>
    
    ```
    		    === g++: Unexpected fails for rv64imafdc lp64d medlow ===
    [Removed some to avoid github's comment size limit]...
    FAIL: g++.old-deja/g++.pt/explicit42.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit42.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit42.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit43.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit43.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit43.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit43.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit50.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit50.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit50.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit50.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit51.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit51.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit51.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit51.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit52.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit52.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit52.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit52.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit53.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit53.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit53.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit53.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit54.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit54.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit54.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit54.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit55.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit55.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit55.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit55.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit56.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit56.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit56.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit56.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit57.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit57.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit57.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit57.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit58.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit58.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit58.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit58.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit59.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit59.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit59.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit59.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit60.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit60.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit60.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit60.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit61.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit61.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit61.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit61.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit62.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit62.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit62.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit62.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit63.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit63.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit63.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit63.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit64.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit64.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit64.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit64.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit65.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit65.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit65.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit65.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit66.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit66.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit66.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit66.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/explicit74.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/explicit74.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/explicit74.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/explicit74.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/expr1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/expr1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/expr1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/expr1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/fntry1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/fntry1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/fntry1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/fntry1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/friend1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/friend1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/friend1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/friend1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/friend10.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/friend10.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/friend10.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/friend10.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/friend12.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/friend12.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/friend12.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/friend12.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/friend13.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/friend13.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/friend13.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/friend13.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/friend4.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/friend4.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/friend4.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/friend4.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/friend5.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/friend5.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/friend5.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/friend5.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/friend6.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/friend6.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/friend6.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/friend6.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/friend7.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/friend7.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/friend7.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/friend7.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/friend8.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/friend8.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/friend8.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/friend8.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/friend9.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/friend9.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/friend9.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/friend9.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/inherit1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/inherit1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/inherit1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/inherit1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/instantiate12.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/instantiate12.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/instantiate12.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/instantiate12.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/local1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/local1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/local1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/local1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/local2.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/local2.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/local2.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/local2.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/local3.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/local3.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/local3.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/local3.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/local4.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/local4.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/local4.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/local4.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/local5.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/local5.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/local5.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/local5.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/local6.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/local6.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/local6.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/local6.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/lookup3.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/lookup3.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/lookup3.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/lookup3.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memclass1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memclass1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memclass1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memclass1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memclass2.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memclass2.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memclass2.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memclass2.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memclass3.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memclass3.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memclass3.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memclass3.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memclass4.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memclass4.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memclass4.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memclass4.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memclass5.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memclass5.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memclass5.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memclass5.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memclass6.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memclass6.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memclass6.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memclass6.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memclass7.C  -std=gnu++14 execution test
    FAIL: g++.old-deja/g++.pt/memclass7.C  -std=gnu++17 execution test
    FAIL: g++.old-deja/g++.pt/memclass7.C  -std=gnu++20 execution test
    FAIL: g++.old-deja/g++.pt/memclass7.C  -std=gnu++98 execution test
    FAIL: g++.old-deja/g++.pt/memtemp28.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memtemp28.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memtemp28.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memtemp28.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memtemp63.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memtemp63.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memtemp63.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memtemp63.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memtemp67.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memtemp67.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memtemp67.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memtemp67.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memtemp69.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memtemp69.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memtemp69.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memtemp69.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memtemp73.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memtemp73.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memtemp73.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memtemp73.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memtemp77.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memtemp77.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memtemp77.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memtemp77.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memtemp96.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memtemp96.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memtemp96.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memtemp96.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/memtemp97.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/memtemp97.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/memtemp97.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/memtemp97.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/mi1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/mi1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/mi1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/mi1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/nttp1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/nttp1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/nttp1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/nttp1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/nttp2.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/nttp2.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/nttp2.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/nttp2.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/overload1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/overload1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/overload1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/overload1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/overload10.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/overload10.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/overload10.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/overload10.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/overload12.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/overload12.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/overload12.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/overload12.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/overload6.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/overload6.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/overload6.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/overload6.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/overload9.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/overload9.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/overload9.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/overload9.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/parms1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/parms1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/parms1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/parms1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/partial1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/partial1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/partial1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/partial1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/partial3.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/partial3.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/partial3.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/partial3.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/partial5.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/partial5.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/partial5.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/partial5.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ptrmem1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ptrmem1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ptrmem1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ptrmem1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ptrmem2.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ptrmem2.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ptrmem2.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ptrmem2.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/recursion2.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/recursion2.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/recursion2.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/recursion2.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ref3.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ref3.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ref3.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ref3.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/sizeof.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/sizeof.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/sizeof.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/sizeof.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/sizeof2.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/sizeof2.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/sizeof2.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/sizeof2.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/sizeof3.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/sizeof3.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/sizeof3.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/sizeof3.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec10.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec10.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec10.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec10.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec11.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec11.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec11.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec11.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec12.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec12.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec12.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec12.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec13.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec13.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec13.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec13.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec15.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec15.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec15.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec15.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec29.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec29.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec29.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec29.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec3.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec3.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec3.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec3.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec30.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec30.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec30.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec30.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec36.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec36.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec36.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec36.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec7.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec7.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec7.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec7.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/spec8.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/spec8.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/spec8.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/spec8.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/static1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/static1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/static1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/static1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/static11.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/static11.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/static11.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/static11.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/static3.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/static3.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/static3.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/static3.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/static8.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/static8.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/static8.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/static8.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/stmtexpr.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/stmtexpr.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/stmtexpr.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/stmtexpr.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/stmtexpr2.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/stmtexpr2.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/stmtexpr2.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/stmtexpr2.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/t16.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/t16.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/t16.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/t16.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/t39.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/t39.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/t39.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/t39.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/t42.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/t42.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/t42.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/t42.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/tiemann2.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/tiemann2.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/tiemann2.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/tiemann2.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp11.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp11.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp11.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp11.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp12.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp12.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp12.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp12.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp13.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp13.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp13.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp13.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp14.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp14.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp14.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp14.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp15.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp15.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp15.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp15.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp16.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp16.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp16.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp16.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp17.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp17.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp17.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp17.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp18.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp18.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp18.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp18.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp20.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp20.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp20.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp20.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp21.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp21.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp21.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp21.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp22.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp22.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp22.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp22.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp23.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp23.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp23.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp23.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp24.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp24.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp24.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp24.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp25.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp25.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp25.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp25.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp27.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp27.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp27.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp27.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp28.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp28.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp28.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp28.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp29.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp29.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp29.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp29.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp30.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp30.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp30.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp30.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp31.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp31.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp31.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp31.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp32.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp32.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp32.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp32.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp33.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp33.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp33.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp33.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp34.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp34.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp34.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp34.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp42.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp42.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp42.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp42.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp47.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp47.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp47.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp47.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp48.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp48.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp48.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp48.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp50.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp50.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp50.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp50.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp6.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp6.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp6.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp6.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp62.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp62.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp62.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp62.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/ttp8.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/ttp8.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/ttp8.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/ttp8.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/typename12.C  -std=gnu++14 execution test
    FAIL: g++.old-deja/g++.pt/typename12.C  -std=gnu++17 execution test
    FAIL: g++.old-deja/g++.pt/typename12.C  -std=gnu++20 execution test
    FAIL: g++.old-deja/g++.pt/typename12.C  -std=gnu++98 execution test
    FAIL: g++.old-deja/g++.pt/typename19.C  -std=gnu++14 execution test
    FAIL: g++.old-deja/g++.pt/typename19.C  -std=gnu++17 execution test
    FAIL: g++.old-deja/g++.pt/typename19.C  -std=gnu++20 execution test
    FAIL: g++.old-deja/g++.pt/typename19.C  -std=gnu++98 execution test
    FAIL: g++.old-deja/g++.pt/typename9.C  -std=gnu++14 execution test
    FAIL: g++.old-deja/g++.pt/typename9.C  -std=gnu++17 execution test
    FAIL: g++.old-deja/g++.pt/typename9.C  -std=gnu++20 execution test
    FAIL: g++.old-deja/g++.pt/typename9.C  -std=gnu++98 execution test
    FAIL: g++.old-deja/g++.pt/using1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/using1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/using1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/using1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/vaarg.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/vaarg.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/vaarg.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/vaarg.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/vbase1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/vbase1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/vbase1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/vbase1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/virtual1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/virtual1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/virtual1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/virtual1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.pt/virtual4.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.pt/virtual4.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.pt/virtual4.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.pt/virtual4.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/980310-1.C  -std=gnu++14 execution test
    FAIL: g++.old-deja/g++.robertl/980310-1.C  -std=gnu++17 execution test
    FAIL: g++.old-deja/g++.robertl/980310-1.C  -std=gnu++20 execution test
    FAIL: g++.old-deja/g++.robertl/980310-1.C  -std=gnu++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb102.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb102.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb102.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb102.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb104.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb104.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb104.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb104.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb111.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb111.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb111.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb111.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb114.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb114.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb114.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb114.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb115.C  -std=gnu++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb115.C  -std=gnu++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb115.C  -std=gnu++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb115.C  -std=gnu++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb116.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb116.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb116.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb116.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb120.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb120.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb120.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb120.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb126.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb126.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb126.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb126.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb14.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb14.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb14.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb14.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb17.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb17.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb17.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb17.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb21.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb21.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb21.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb21.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb26.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb26.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb26.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb26.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb31.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb31.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb31.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb31.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb33.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb33.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb33.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb33.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb41.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb41.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb41.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb41.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb46.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb46.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb46.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb46.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb50.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb50.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb50.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb50.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb54.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb54.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb54.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb54.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb59.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb59.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb59.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb59.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb65.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb65.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb65.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb65.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb66.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb66.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb66.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb66.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb73.C  -std=gnu++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb73.C  -std=gnu++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb73.C  -std=gnu++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb73.C  -std=gnu++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb77.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb77.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb77.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb77.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb84.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb84.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb84.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb84.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb88.C  -std=gnu++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb88.C  -std=gnu++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb88.C  -std=gnu++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb88.C  -std=gnu++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb91.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb91.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb91.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb91.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb95.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb95.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb95.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb95.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eb97.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eb97.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eb97.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eb97.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull-cast.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull-cast.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull-cast.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull-cast.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull-oldcast.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull-oldcast.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull-oldcast.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull-oldcast.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/ebnull.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-1.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-1.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-1.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-1.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-2.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-2.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-2.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-2.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-3.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-3.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-3.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-3.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-4.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-4.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-4.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-4.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-5.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-5.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-5.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/eh990323-5.C  -std=c++98 execution test
    FAIL: g++.old-deja/g++.robertl/ice990323-2.C  -std=c++14 execution test
    FAIL: g++.old-deja/g++.robertl/ice990323-2.C  -std=c++17 execution test
    FAIL: g++.old-deja/g++.robertl/ice990323-2.C  -std=c++20 execution test
    FAIL: g++.old-deja/g++.robertl/ice990323-2.C  -std=c++98 execution test
    FAIL: g++.target/riscv/call-with-empty-struct-float.C execution test
    FAIL: g++.target/riscv/call-with-empty-struct-int.C execution test
    
                   ========= Summary of gcc testsuite =========
                                | # of unexpected case / # of unique unexpected case
                                |          gcc |          g++ |     gfortran |
     rv64imafdc/  lp64d/ medlow |19208 /  3644 | 9845 /  2462 |      - |
    make: *** [Makefile:980: report-gcc-newlib] Error 1
    ```
</details>

CC: @kito-cheng @cmuellner @vineetgarc

`make report-linux` results:
```
               ========= Summary of gcc testsuite =========
                            | # of unexpected case / # of unique unexpected case
                            |          gcc |          g++ |     gfortran |
 rv64imafdc/  lp64d/ medlow |19620 /  3743 | 9929 /  2481 |17972 /  3029 |
make: *** [Makefile:987: report-gcc-linux] Error 1
```

--- a/Makefile.in
+++ b/Makefile.in
@@ -178,7 +178,7 @@ report-gdb: report-gdb-@default_target@
 ifeq ($(SIM),qemu)
 SIM_PATH:=$(srcdir)/scripts/wrapper/qemu:$(srcdir)/scripts
 SIM_PREPARE:=PATH="$(SIM_PATH):$(INSTALL_DIR)/bin:$(PATH)" RISC_V_SYSROOT="$(SYSROOT)"
-SIM_STAMP:= stamps/build-qemu
+SIM_STAMP:= stamps/build-qemu stamps/install-python-package
 else
 ifeq ($(SIM),spike)
 # Using spike simulator.
@@ -870,6 +870,10 @@ stamps/build-pk64: $(PK_SRCDIR) stamps/build-gcc-newlib-stage2
 	$(MAKE) -C $(notdir $@)
 	cp $(notdir $@)/pk $(INSTALL_DIR)/$(NEWLIB_TUPLE)/bin/pk64
 	mkdir -p $(dir $@)
+	date > $@
+
+stamps/install-python-package:
+	python3 -m pip install --user pyelftools
 	date > $@
 
 stamps/build-qemu: $(QEMU_SRCDIR) $(QEMU_SRC_GIT)

